datacats/error.py

```diff
@@ -4,7 +4,10 @@
 class DatacatsError(Exception):
 
     def __init__(self, message, format_args=(), parent_exception=None):
-        self.message = message
+        try:
+            self.message = message.format(format_args)
+        except (ValueError, KeyError):
+            self.message = message
```

Comment:
You are right, it should be `*format_args`, thanks.

ValueError and KeyError are raised for this two cases correspondingly:

```python
"I like {".format(fruit=bananas) # ValueError
"I like {vegetable} ".format(fruit=bananas) # KeyError
```